### PR TITLE
[READY-TO-REVIEW] Update Coming_Next.md for Templates

### PR DESCRIPTION
**Pull request by jarusso**

### CODE REVIEW

**Comment by jarusso** [comment]
@miguelestrada review and merge please

File: Coming_Next.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Here’s the stuff we’re working on next for you and IBM Watson Work Services.
 
 | Coming Next   |
 | :-------------|
+| **Templates.** We're adding **Templates** for you to create custom experiences for teams. It's `EXPERIMENTAL` but you can check it out here, [Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_space_template_info.md). We have [Create a Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_create_space_template.md), [Space Template Info](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_space_template_info.md), [Get Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_get_space_template.md), and [Delete a Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_delete_space_template.md).              |
```
Comment (miguelestrada) [inline Coming_Next.md]: @jarusso here is the main info doc for templates.  Can we link to it as first link?
https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_space_template_main.md

Comment (miguelestrada) [reply Coming_Next.md]: If you want the complete list to be displayed in the `Coming Next` section then this one should be included as well:

https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_get_offering_space_templates.md

**Review by miguelestrada — CHANGES_REQUESTED** [review]

**Comment by jarusso** [comment]
@miguelestrada changes made

File: Coming_Next.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Here’s the stuff we’re working on next for you and IBM Watson Work Services.
...
+| **Templates.** We're adding **Templates** for you to create custom experiences for teams. It's `EXPERIMENTAL` but you can check it out here, [Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_space_template_main.md). We have [Create a Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_create_space_template.md), [Space Template Info](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_space_template_info.md), [Get Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_get_space_template.md), and [Delete a Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_delete_space_template.md). For templates associated with specific offerings, we have [Get Offering Space Templates](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_get_offering_space_templates.md).              |
```
Comment (miguelestrada) [inline Coming_Next.md]: Missing one.
https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_get_my_space_templates.md

**Review by miguelestrada — CHANGES_REQUESTED** [review]
https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_get_my_space_templates.md

**Comment by jarusso** [comment]
good catch, fixed that item @miguelestrada

File: Coming_Next.md [hunk 3]
```diff
@@ -9,6 +9,7 @@ Here’s the stuff we’re working on next for you and IBM Watson Work Services.
 
 | Coming Next   |
 | :-------------|
+| **Templates.** We're adding **Templates** for you to create custom experiences for teams. It's `EXPERIMENTAL` but you can check it out here, [Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_space_template_main.md). We have [Create a Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_create_space_template.md), [Space Template Info](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_space_template_info.md), [Get Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_get_space_template.md), [Get My Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_get_my_space_templates.md) and [Delete a Space Template](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_delete_space_template.md). For templates associated with specific offerings, we have [Get Offering Space Templates](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_get_offering_space_templates.md).              |
```
Comment (miguelestrada) [inline Coming_Next.md]: One minor thing.... can we pluralize this?  It returns a collection not a template....  it is the analog to getting a list of all the apps you have created.... but for templates...  

>Get My Space Template

to

>Get My Space Templates

**Review by miguelestrada — CHANGES_REQUESTED** [review]

**Review by miguelestrada — APPROVED** [review]

**Pull request merged**